.sass-lint.yml

```diff
@@ -11,7 +11,7 @@ rules:
   no-url-protocols: 0
   class-name-format:
     - 1
-    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,2}$
+    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,4}$
     - convention-explanation: 'Classes should be camelCase and separated by no more than 5 hyphens'
```

Comment:
@amberRrucker Spoke with @SyntaxStacks about this issue. Can we create a class called `.faBefore-white`, or something like that and just add it to these references in the footer?   That will get rid of the this issue in this context.  We'll still have to solve it elsewhere, but that at least helps us not have it hold up this PR.